README.md

```diff
@@ -46,7 +66,176 @@ Pipelines:
+
+````text
+cd $JF_PRODUCT_DATA_INTERNAL
+wget https://raw.githubusercontent.com/jfrog/log-analytics-datadog/master/fluent.conf.missioncontrol
+````
+
+Override the match directive(last section) of the downloaded `fluent.conf.missioncontrol` with the details given below
+
+```
+<match jfrog.**>
+  @type datadog
+  @id datadog_agent_jfrog_missioncontrol
+  api_key API_KEY
+  include_tag_key true
+  dd_source fluentd
+</match>
+```
+
+_**required**_: ```API_KEY``` is the apiKey from Datadog
+
+```dd_source``` attribute is set to the name of the log integration in your logs in order to trigger the integration automatic setup in datadog.
+
+```include_tag_key``` defaults to false and it will add fluentd tag in the json record if set to true
+
+After saving the configuration file, please proceed to the installation part - [Fluentd Installation](#fluentd-installation)
+
+### Configuration steps for Distribution
+
+Download the distribution fluentd configuration file to a directory the user has permissions to write, such as the $JF_PRODUCT_DATA_INTERNAL locations discussed above in the [Environment Configuration](#environment-configuration) section.
+
+````text
+cd $JF_PRODUCT_DATA_INTERNAL
+wget https://raw.githubusercontent.com/jfrog/log-analytics-datadog/master/fluent.conf.distribution
+````
+
+Override the match directive(last section) of the downloaded `fluent.conf.distribution` with the details given below
+
+```
+<match jfrog.**>
+  @type datadog
+  @id datadog_agent_jfrog_distribution
+  api_key API_KEY
+  include_tag_key true
+  dd_source fluentd
+</match>
+```
+
+_**required**_: ```API_KEY``` is the apiKey from Datadog
+
+```dd_source``` attribute is set to the name of the log integration in your logs in order to trigger the integration automatic setup in datadog.
+
+```include_tag_key``` defaults to false and it will add fluentd tag in the json record if set to true
+
+After saving the configuration file, please proceed to the installation part - [Fluentd Installation](#fluentd-installation)
+
+### Configuration steps for Pipelines
+
+Download the pipelines fluentd configuration file to a directory the user has permissions to write, such as the $JF_PRODUCT_DATA_INTERNAL locations discussed above in the [Environment Configuration](#environment-configuration) section.
+
+````text
+cd $JF_PRODUCT_DATA_INTERNAL
+wget https://raw.githubusercontent.com/jfrog/log-analytics-datadog/master/fluent.conf.pipelines
+````
+
+Override the match directive(last section) of the downloaded `fluent.conf.pipelines` with the details given below
+
+```
+<match jfrog.**>
+  @type datadog
+  @id datadog_agent_jfrog_pipelines
+  api_key API_KEY
+  include_tag_key true
+  dd_source fluentd
+</match>
+```
+
+_**required**_: ```API_KEY``` is the apiKey from Datadog
```

Comment:
Add steps and screenshots on how to get this from DataDog. It was indeed confusing - because of how datadog manages their UX.

Reply:
Adding a link to datadog which has steps to get API Key